fdbserver/Status.actor.cpp

```diff
@@ -781,6 +781,8 @@ ACTOR static Future<JsonBuilderObject> processStatusFetcher(
 				// Map the address of the worker to the error message object
 				tracefileOpenErrorMap[traceFileErrorsItr->first.toString()] = msgObj;
 			} catch (Error& e) {
+				if (e.code() == error_code_actor_cancelled)
```

Comment:
Can we add a brackets around this block? (Same below)

Reply:
Done.